auto_round/mllm/__init__.py

```diff
@@ -16,4 +16,4 @@
 from .template import Template, get_template, TEMPLATES
 from .autoround_mllm import AutoRoundMLLM
 from ..utils import LazyImport
-from .eval import mllm_eval
\ No newline at end of file
+from .eval import mllm_eval, lmms_eval
```

Comment:
make sure that the tune/ inference work fine when  these two eval library have not been installed